_test/common_functions/FakeFAccess.m

```diff
@@ -0,0 +1,62 @@
+classdef FakeFAccess < sqw_binfile_common
+% A fake file access class that can return data as if from an sqw file
+
+properties
+    fake_data = [];
+    closed = false;
+end
+
+methods
+
+    function obj = FakeFAccess(data)
+        obj.fake_data = data;
+        obj.npixels_ = size(data, 2);
+    end
+
+    function data = get_pix(obj, varargin)
+        if nargin > 2
+            [pix_lo, pix_hi] = varargin{:};
+        else
+            pix_lo = 1;
+            pix_hi = size(obj.fake_data, 2);
+        end
+        try
+            data = obj.fake_data(:, pix_lo:pix_hi);
+        catch ME
+            switch ME.identifier
+            case 'MATLAB:badsubscript'
+                error('FAKEFACCESS:get_pix', 'End of fake file reached');
+            otherwise
+                rethrow(ME);
+            end
+        end
+    end
+
+    function new_obj = upgrade_file_format(obj)
+        new_obj = [];
```

Comment:
not sure how the actual functionality - returning empty matrix - fits with the title. Maybe a comment?

Reply:
`FakeFAccess.m` has been copied, unchanged, from `master`

Reply:
If I remember correctly, this was an abstract method that needed implementing (or construction would raise an error). Coming back to it, it may have been best to raise a "Not implemented" error. In any case, this method is unused.